### PR TITLE
Move Windows from AppVeyor to GitHub Actions

### PR DESCRIPTION
**Pull request by IntelOrca**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,8 +73,8 @@ OpenLoco requires original files of Chris Sawyer's Locomotion to play. It can be
 ### Windows:
 1. Check out the repository. This can be done using [GitHub Desktop](https://desktop.github.com) or [other tools](https://help.github.com/articles/which-remote-url-should-i-use).
 2. Install dependencies using [vcpkg](https://github.com/microsoft/vcpkg) or use the [nuget package](https://github.com/OpenLoco/Dependencies/releases).
```
Comment (AaronVanGeffen) [inline README.md:75]: Does this line need to remain, now that `/t:restore` has been added as a flag for msbuild?

Comment (AaronVanGeffen) [reply README.md:75]: Just so we don't forget: the AppVeyor badge should be removed from the readme as well.

**Comment by IntelOrca** [comment]
image

CI build works!

**Comment by AaronVanGeffen** [comment]
Nice! Looks like it's missing the branch/tag info though. Could you look into that?

**Comment by IntelOrca** [comment]
image

**Pull request merged**